crates/sui-indexer/migrations_v2/2023-08-19-044026_transactions/up.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE transactions (
+    tx_sequence_number          BIGINT       PRIMARY KEY,
```

Comment:
Is `tx_sequence_number` alone the primary key? I thought it was local to a particular checkpoint.

Reply:
this is a global index that is (partially) used for pagination

Reply:
Can you always identify the transactions in a checkpoint by a range of sequence numbers?  If so, then this is probably fine.  Otherwise, this will complicate implementing consistency across all queries (like trying to consistently paginate transactions sent by an address, which needs to be restricted to the transactions at or before the latest checkpoint indexed at the time the first page was queried).

In GraphQL, cursors will be opaque, and will probably be represented by Base64 encoding compound keys, like `base64("c:<checkpoint_seq>;t:<tx_seq>;e:<ev_seq>")` for an event cursor.

Reply:
Yes this number is intended to be the total order sequence of the transactions, and given a checkpoint you should be able to concretely tell which range of sequence numbers apply to it